Turn off AQUA AIRS channel 305 after 20220111_12z

gmao-msienkie opened the pull request:
Channel is becoming noisy starting 2022/01/11 18z
(will be non-zero diff after 2022/01/11 12z)

gmao-msienkie committed:
Turn off AQUA AIRS channel 305 after 20220111_12z
Channel is becoming noisy

jing-guo reviewed (commented):
It seems this is is a single channel (305) change, **only** on the original line 563.  
```
aqua  20160928 120000  21001231 240000  airs    9  257  261  262  267  272  295  299       305  310 #   100
```
If so, it might be better to make change on that single line only, to
```
aqua  20160928 120000  20220111 120000  airs    9  257  261  262  267  272  295  299       305  310 #   100
aqua  20220211 180000  21001231 240000  airs    8  257  261  262  267  272  295  299            310 #   100 - ch 305 became noisy
```
Note, the old channel count 9 should be changed to 8.  I hope this way might help to understand the content better.

gmao-msienkie committed:
Correct error in AIRS update

Correct error in AIRS update - number of channels on line 100 is now 8, not 9

gmao-msienkie commented:
Thanks @jing-guo for the reminder about changing the number of channels.  I corrected this in the files I passed to Rob for the operational systems but forgot to fix in in the file I checked in.
I have fixed this now.

jing-guo reviewed (approved):
The fix looks good.  

However, I don't see the reason to modify the date ranges of all channels in the full instrument block to fix only a single channel.  It distracts reviewer's attention from the real change.

rtodling committed:
Merge branch 'develop' into feature/msienkie/airs_ch305_noisy
